File: reverse.js
var arr = [1, 3, 35, 6, 7, 8];
function ReverseArray(arr) {
  var len = arr.length;
  for (var i = 1; i <= len; i++) {
    len1 = len - i;
    console.log(arr[len1]);
  }
}

ReverseArray(arr);

File: reverse.go
package main

import "fmt"

func reverse(arr [3]int) []int {
	TotalLength := len(arr) //finding length of the array
	// Debugging
	// fmt.Println(TotalLength)
	var reverse_array []int //creating the array to return reverse order of arry
	for i, _ := range arr {
		i++                                            // make i value one to index bound problem got solved
		from_back := TotalLength - i                   //revesing the index
		reverse := arr[from_back]                      //finally reversing all values
		reverse_array = append(reverse_array, reverse) //appending data one by one to new array/slice
	}
	return reverse_array
}
func main() {
	var arr = [3]int{121, 223, 454545}
	rev := reverse(arr)
	fmt.Println(rev)
}
